### PR TITLE
Merge Umcs 45 to develop

### PR DESCRIPTION
**Pull request by DerAusHeidelberg**

### CODE REVIEW

**Review by sixer1182 — COMMENTED** [review]
I made some comments.
Please check them and give me feedback if it is no problem.

**Review by Ryouzanpaku — COMMENTED** [review]
- Please also update the changelog: "Add email property to payment type `card` to meet 3Ds2.x regulations."
- Is there a test fetching a card and checking for the fetched email? If not it would be good to add or update the existing `testFetchCardType()`

File: src/test/java/com/unzer/payment/business/paymenttypes/CardTest.java [hunk 1]
```diff
@@ -37,119 +38,193 @@
...
+    @Test
+    public void testCreateCardWithMerchantNotPCIDSSCompliant() throws HttpCommunicationException {
+        ThrowingRunnable createCardWithMerchantNotPCIDSSCompliant = new ThrowingRunnable() {
+            @Override
+            public void run() throws Throwable {
+                Card card = new Card("4444333322221111", "03/20");
+                card.setCvc("123");
+                Unzer unzer = new Unzer(new HttpClientBasedRestCommunication(), privateKey3);
+                card = unzer.createPaymentType(card);
+            }
```
[the remark this answers is not in the document]
Comment (DerAusHeidelberg) [reply src/test/java/com/unzer/payment/business/paymenttypes/CardTest.java:50]: No, it is still thrown and test ist passing.

File: src/main/java/com/unzer/payment/communication/json/JsonCard.java [hunk 2]
```diff
@@ -92,4 +93,12 @@ public String getBrand() {
 	public void setBrand(String brand) {
 		this.brand = brand;
 	}
+
+	public String getEmail() {
+		return email;
+	}
+
+	public void setEmail(String email) {
```
[the remark this answers is not in the document]
Comment (DerAusHeidelberg) [reply src/main/java/com/unzer/payment/communication/json/JsonCard.java:101]: as `String` can be null in Java this is valid with the spec

File: src/main/java/com/unzer/payment/paymenttypes/Card.java [hunk 3]
```diff
@@ -169,4 +180,12 @@ public CardDetails getCardDetails() {
 	public void setCardDetails(CardDetails cardDetails) {
 		this.cardDetails = cardDetails;
 	}
+
+	public String getEmail() {
+		return email;
+	}
+
+	public void setEmail(String email) {
```
[the remark this answers is not in the document]
Comment (DerAusHeidelberg) [reply src/main/java/com/unzer/payment/paymenttypes/Card.java:188]: see above

**Review by Ryouzanpaku — APPROVED** [review]
LGTM

**Pull request merged**